0018-connector-risk-mitigations/0018-connector-risk-mitigations.md

```diff
@@ -17,25 +17,28 @@ The main risk connectors face in Interledger is being unable to fulfill the inco
 Possible mitigations include:
 
 * **Connecting Reliable Ledgers** - Connectors should choose the ledgers they connect in part based on the reliability and speed of fulfillment notifications and the time the ledgers require to process fulfillments once they are submitted
-* **Adjusting Transfer Expiry Window** - Connectors determine the window of time they require between when the incoming and outgoing transfers expire. Connectors should set this window such that they are confident that they will be able to deliver the fulfillment in time even if the outgoing transfer is executed at the last possible moment.
 * **Denial of Service Protections** - Connectors should take standard [Denial of Service](https://en.bitcoin.it/wiki/Weaknesses#Denial_of_Service_.28DoS.29_attacks) attack prevention steps to prevent attackers from overloading the connectors' servers with network traffic
 * **Prioritizing Fulfillments** - Connectors should prioritize fulfilling transfers over preparing new ones or responding to quote requests and may even have separate processes or machines responsible for those different behaviors
 * **Redundant Instances** - Connectors should run redundant processes and machines to increase the difficulty of interfering with their processing
 * **Avoiding Public Endpoints** - Connectors may reduce attack vectors by using communication channels and protocols that do not entail having a publicly-accessible server (for example using [WebRTC](https://webrtc.org/) or [Virtual Private Networks](https://en.wikipedia.org/wiki/Virtual_private_network) to communicate with peers
 
+Note that newer unconditional payment channels in ILPv4 do not require longer timeouts while enabling peers to rebalance frequently and minimize their bilateral trust.
```

Comment:
I think we could expand on this note to explain how ILPv4 changed the whole model.

The ledger is no longer in the flow so the nature of the protocol is different. In the past, ILP worked best over ledgers that could prepare and fulfill transfers quickly as this time was additive along the path of a payment.

With ILPv4 we prefer ledgers that can simple complete a transfer quickly and inexpensively so that we can do frequent bilateral settlements and keep the settlement risk between peers low.

Reply:
Very few people knew what ILP was when we were talking about ILPv1 so it feels like it might be better to take out everything that's v1-related and just focus on v4.